src/helpers/edit.js

```diff
@@ -1,5 +1,18 @@
 import { findIndex, map } from 'lodash'
 
+function isProjectStaffed(projectRow) {
+  const projectName = projectRow.project
```

Comment:
can you do `const {project:projectName} = projectRow `instead ?

Reply:
Done!